pandas/tests/io/parser/test_dtypes.py

```diff
@@ -192,7 +192,7 @@ def test_categorical_dtype_utf16(all_parsers, csv_dir_path):
     pth = os.path.join(csv_dir_path, "utf16_ex.txt")
     parser = all_parsers
     encoding = "utf-16"
-    sep = ","
+    sep = "\t"
```

Comment:
why did this change?

Reply:
This test failed for some reason. So when I investigated this I noticed that the separator in this file is a tab rather than a ,. So I changed this and the test passed. I am not sure what caused this, but I think that the test was wrong to begin with and there is some weird feature interaction that caused the failure.

Reply:
Hmm yea very strange. I guess since there isn't a comma in this file at all it just reads every line into a single column....